.rubocop.yml

```diff
@@ -42,6 +42,15 @@ Style/IpAddresses:
   Exclude:
     - spec/rubocop/cop/style/ip_addresses_spec.rb
 
+Style/FetchEnvVar:
+  AllowedVars:
+    - HOME
+    - RUBOCOP_CACHE_ROOT
+    - XDG_CACHE_HOME
+    - XDG_CONFIG_HOME
+    - CIRCLE_STAGE
+    - CI
```

Comment:
How were these 6 variable names chosen as exceptions to the new rule?

Reply:
Thanks for pointing it out. They are the environment variables used in this project. I listed them here to get opinions on how these should be. I think we can replace almost all `ENV[...]` in this project with `ENV.fetch(..., nil)`, except that [`ENV['XDG_CACHE_HOME']` and `ENV['HOME']` here](https://github.com/rubocop/rubocop/blob/a4f01948b7e83cec2678d359dfda426da89cce86/lib/rubocop/result_cache.rb#L74-L84) probably should be replaced with `ENV.fetch(...)` without the default values. Do you have any opinions?

Reply:
I added a commit to show how the code will be.
https://github.com/rubocop/rubocop/pull/10502/commits/190111747122d83e49123b7e1279aeb41a329ebf